### DISCUSSION TITLE
Discrepancies in Tree Topology with Different Models in IQ-TREE

### DISCUSSION THREAD

**Discussion by adriadieguez** [iqtree/iqtree2#222] [Q&A] [answered]
Hi everyone,

I'm encountering an issue with tree topology discrepancies when using different models in IQ-TREE and I'm looking for some suggestions.

Considering a quartet with branch lengths 0.1, for example, **(Leaf_1:0.1,Leaf_2:0.1,(Leaf_3:0.1,Leaf_4:0.1):0.1)**,  I've used an alignment simulator to generate alignments given this topology and these branch lengths.

Then, I have provided the simulated FASTA file and the tree topology (without branch lengths) as inputs to IQ-TREE to estimate the branch lengths of the tree. When I use the **GTR model (012345)**, I obtain very accurate results for branch length estimation, and the correct topology is returned. For example: **(Leaf_1:0.099,Leaf_2:0.101,(Leaf_3:0.101,Leaf_4:0.096):0.102)**;

 However, using the **12.12 model,** I notice that IQ-TREE returns different topologies, such as: 

-  **((Leaf_1:0.100,Leaf_2:0.094):0.055,(Leaf_3:0.102,Leaf_4:0.100):0.049):0.000** or 
-  **((Leaf_1:0.095,(Leaf_3:0.098,Leaf_4:0.099):0.104):0.088,Leaf_2:0.000):0.000**;

 I also noticed that the output with the GTR model (012345) gives an unrooted tree, whereas the 12.12 model always returns a rooted tree and places a root. Could this be contributing to the observed discrepancies?

Any help or guidance would be greatly appreciated.  I'm particularly interested in understanding why the 12.12 model leads to different topologies. Thank you in advance!

Best regards,

Adrià

**Comment by lccra** [comment, marked as answer]
12.12 is a non-reversible model -> see the tutorial on [Rooting phylogenetic trees](http://iqtree.org/doc/Rootstrap), specifically the "Inferring rooted trees without outgroup" part

**Reply by roblanf** [reply]
Just to add to this, once you account for the rooting issues, there are no discrepancies in the trees you have provided. The topologies and branch lengths are as expected.
